README.rst

```diff
@@ -1,54 +1,48 @@
-.. image:: https://travis-ci.com/plone/training.svg?branch=master
-    :target: https://travis-ci.com/plone/training
-
 ========
 Training
 ========
 
-`Training <https://github.com/plone/training>`_ is a collection of different trainings,
+`training.plone.org <https://training.plone.org>`_ is a collection of different trainings,
 developed and created by the Plone Community.
 
-For a HTML version, please browse to our `Training Website <https://training.plone.org/5/>`_.
 
 Overview
 ========
 
-Different Plone Trainings:
-
-- `How to work with content and manage site settings in Plone. <https://training.plone.org/5/>`_
-- `How to get your Plone site up and running. <https://training.plone.org/5/deployment/index.html>`_
-- `How to develop customized solutions with Plone. <https://training.plone.org/5/mastering_plone/index.html#mastering-plone-label>`_
-- `How to add enterprise grade search to your Plone site. <https://training.plone.org/5/solr-training/index.html>`_
-- `How to style your Plone site. <https://training.plone.org/5/theming/index.html>`_
-- `Create custom content types, a design for a website, layouts for homepages and content types, and custom application logic. All in the browser. <https://training.plone.org/5/ttw/index.html>`_
+Beside the trainings you can find information on `Plone documentation <https://docs.plone.org/>`_
 
-Documentation
-=============
+`How to work with content and manage site settings in Plone. <https://docs.plone.org/working-with-content/>`_
```

Comment:
Remove, or change to a specific training, such as ["Through-The-Web" Plone Customization](https://training.plone.org/5/ttw/index.html).